packaging/pkg_helpers.bash

```diff
@@ -170,6 +170,7 @@ setup_pip_pytorch_version() {
     fi
   else
     pip_install "torch==$PYTORCH_VERSION$PYTORCH_VERSION_SUFFIX" \
+      -f https://download.pytorch.org/whl/torch_test.html \
```

Comment:
This should actually be set to `https;//download.pytorch.org/whl/test/torch_test.html`

```suggestion
      -f https://download.pytorch.org/whl/test/torch_test.html \
```